rest-api/api-docs/channels.json

```diff
@@ -1898,6 +1898,54 @@
 					]
 				}
 			]
+		},
+		{
+			"path": "/channels/{channelId}/transfer_progress",
+			"description": "Inform the channel that the transfer is in progress.",
+			"operations": [
+				{
+					"httpMethod": "POST",
```

Comment:
Please add between "httpMethod" and "summary"...
"since": [
	"22.3.0",
	"21.8.0",
	"20.13.0"
],